TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -14,6 +14,87 @@ import UIKit
     
     static let kEmployeesOrderHasChanged = "EmployeesOrderHasChanged"
     
+    static let kOrganizationHasChanged = "OrganizationHasChanged"
+    
+    @IBAction func fetchOrganizations()
+    {
+        RequestManager.fetchOrganizations { (response) in
+        
+        var newOrganization:FFOrganization
+        
+        var newEmployee:FFEmployee
+            
+        let countOfOrganizations = DatabaseController.requestResults(for: nil, sortDescriptors: nil, entity: "FFOrganization")
+        print(countOfOrganizations?.count as Any)
+        
+        if (countOfOrganizations?.count)! <= 1
+        {
+        let organizations = response["organizations"]
+        
+        for jsonObjects in organizations as! [AnyObject]
+            {
+                let organizationName = jsonObjects["name"] as! String
+                
+                let organizationEmployees = jsonObjects["employees"] as! [[String : AnyObject]]
+                
+                print("org name = \(organizationName)  employess = \(organizationEmployees)")
+                
+                newOrganization = NSEntityDescription.insertNewObject(forEntityName: "FFOrganization", into: DatabaseController.sharedInstance().context) as! FFOrganization
```

Comment:
Move logic of creating organization from dictionary to Organization class. For that create a method
static func create(fromJSON json: [String : AnyObject])